Codigo/ControlTemperaturaVersionEnC/funciones.h

```diff
@@ -15,18 +15,18 @@
 #define CEROKELVIN -273.15
 #define T0TERMISTOR 298.15 //kelvin
 #define UNOSOBRET0TERMISTOR 0.003354016435
-#define UNOSOBREBETA0 0.0002958579882
-#define BETATERMISTOR 3982 //kelvin
+#define UNOSOBREBETA0 0.0002911208151
+#define BETATERMISTOR 3435 //kelvin
 #define R0TERMISTOR 10000 //10k
 #define SALIDA_MAXIMA_MODO_CALOR 128
 #define SALIDA_MINIMA_MODO_CALOR 0
-#define SALIDA_MAXIMA_MODO_FRIO 255
-#define SALIDA_MINIMA_MODO_FRIO 0
+#define SALIDA_MAXIMA_MODO_FRIO 0
+#define SALIDA_MINIMA_MODO_FRIO -255
 #define TEMPERATURA_MAXIMA 49.0
-#define TEMPERATURA_MINIMA -10.0
-#define DELAYVALUE 100.00
+#define TEMPERATURA_MINIMA 5.0
+#define DELAY_VALUE 100.00
 #define CANTIDAD_COMBINACIONES_PWM_DIFFTEMP 4
-#define TOLERANCIA_DELAY 0.05
+#define TOLERANCIA_DELAY 0.08
 #define TAU 0.63
```

Comment:
remove the followings defines TAU,TENSION_REFERENCIA_OPAMPS, RESISTENCIA_R3, CANTIDAD_COMBINACIONES_PWM_DIFFTEMP,RESISTENCIA_R8,RESISTENCIA_R9,RESISTENCIA_R10